### ISSUE TITLE
Slow application startup

### ISSUE THREAD

**Issue by konstantin-popov** [telegramdesktop/tdesktop#9305]
After update to 2.4.7 (I think) application became very slow to start (20-30 seconds). I can see this time gap in logs but it's not clear for me what is the cause.

UPD: Also notices some error messages in console just before the window opens:
```
error: : cannot open
error: : cannot open
error: : cannot open
```

### Steps to reproduce
1. Start telegram-desktop
2. Wait for the application window to show up

### Expected behaviour
App window opens in few seconds

### Actual behaviour
App window opens in about 30 seconds

### Configuration
**Operating system:** Fedora 32

**Version of Telegram Desktop:** 2.4.7 (but)

**Installation source (Linux Only)** - the official website / GitHub releases / flatpak / snap / distribution package: rpmfusion package for Fedora 32

**Used theme**: default

<details><summary><b>Logs</b>:</summary>
[2020.11.19 12:59:38] Launched version: 2004007, install beta: [FALSE], alpha: 0, debug mode: [TRUE]
[2020.11.19 12:59:38] Executable dir: /usr/bin/, name: telegram-desktop
[2020.11.19 12:59:38] Initial working dir: /home/pconstant/
[2020.11.19 12:59:38] Working dir: /home/pconstant/.local/share/TelegramDesktop/
[2020.11.19 12:59:38] Command line: telegram-desktop -debug
[2020.11.19 12:59:38] Executable path before check: /usr/bin/telegram-desktop
[2020.11.19 12:59:38] Logs started
[2020.11.19 12:59:38] Launcher filename: telegramdesktop.desktop
[2020.11.19 13:00:03] Connecting local socket to /run/user/1000/8c7c4e90550824aa08fe36bfef37b053-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2020.11.19 13:00:03] This is the only instance of Telegram, starting server and app...
[2020.11.19 13:00:03] Moved logging from '/home/pconstant/.local/share/TelegramDesktop/log_start0.txt' to '/home/pconstant/.local/share/TelegramDesktop/log.txt'!
[2020.11.19 13:00:03] Primary screen DPI: 96.1263
[2020.11.19 13:00:03] App Info: reading settings...
[2020.11.19 13:00:03] App Info: reading encrypted settings...
[2020.11.19 13:00:03] Lang Info: Loaded cached, keys: 2800
[2020.11.19 13:00:03] Getting DE, XDG_CURRENT_DESKTOP: 'KDE'
[2020.11.19 13:00:03] Getting DE, DESKTOP_SESSION: '/usr/share/xsessions/plasma'
[2020.11.19 13:00:03] Getting DE, KDE_SESSION_VERSION: '5'
[2020.11.19 13:00:03] DE: KDE5
[2020.11.19 13:00:03] OpenAL Logging Level: (not set)
[2020.11.19 13:00:03] Audio Playback Devices: Встроенное аудио Аналоговый стерео
[2020.11.19 13:00:03] Audio Playback Default Device: Встроенное аудио Аналоговый стерео
[2020.11.19 13:00:03] Audio Capture Devices: Встроенное аудио Аналоговый стерео;Monitor of Встроенное аудио Аналоговый стерео
[2020.11.19 13:00:03] Audio Capture Default Device: Встроенное аудио Аналоговый стерео
[2020.11.19 13:00:03] System tray available: [TRUE]
[2020.11.19 13:00:03] Not using D-Bus global menu.
[2020.11.19 13:00:03] Using Unity launcher counter.
[2020.11.19 13:00:03] App Info: reading accounts info...
[2020.11.19 13:00:03] App Info: reading encrypted info...
[2020.11.19 13:00:03] App Info: reading map...
[2020.11.19 13:00:03] App Info: reading encrypted map...
[2020.11.19 13:00:03] App Info: reading encrypted user settings...
[2020.11.19 13:00:03] App Info: encrypted user settings read.
[2020.11.19 13:00:03] App Info: reading encrypted mtp data...
[2020.11.19 13:00:03] MTP Info: read keys, current: 6, to destroy: 0
[2020.11.19 13:00:03] Map read time: 6
[2020.11.19 13:00:03] App Info: reading encrypted mtp config...
[2020.11.19 13:00:03] Export Info: Destroy top bar by controller removal.
</details>

**Comment by ilya-fedin** [comment]
Already fixed in 2.4.20

**Comment by github-account1111** [comment]
Not for me it's not. Sometimes it's fast sometimes it takes a while. The window itself opens quickly, but I'm stuck looking at a blank page for several seconds before the UI and chats get rendered. That's on an NVME SSD.
